packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Btc/ImportedAddresses/index.tsx

```diff
@@ -16,90 +17,87 @@ class ImportedAddressesContainer extends React.Component<Props> {
   }
 
   handleClickVerify = () => {
-    this.props.modalActions.showModal('VERIFY_MESSAGE_AIRDROP', {
+    this.props.modalActions.showModal('VERIFY_MESSAGE_MODAL', {
       origin: 'SettingsPage'
     })
   }
 
-  handleShowPriv = address => {
+  handleShowPriv = (address) => {
     this.props.modalActions.showModal('SHOW_BTC_PRIVATE_KEY_MODAL', {
       addr: address.addr,
       balance: address.info.final_balance,
       origin: 'SettingsPage'
     })
   }
 
-  handleSignMessage = address => {
+  handleSignMessage = (address) => {
     this.props.modalActions.showModal('SIGN_MESSAGE_MODAL', {
       address: address.addr,
       origin: 'SettingsPage'
     })
   }
 
-  handleEditLabel = address => {
+  handleEditLabel = (address) => {
     this.props.componentActions.editImportedAddressLabel(address.addr)
   }
 
-  handleToggleArchived = address => {
-    let isArchived = address.tag === 2
+  handleToggleArchived = (address) => {
+    const isArchived = address.tag === 2
```

Comment:
maybe for later would be helpful to add comment what number 2 means :)

Reply:
I have no idea what it means. It was here way before I did, and there are no docs for it.

I'll ask around to see what's it, but this is not relevant for this PR. I just changed it from `let` to `const` because the linter asked me to